sky/utils/schemas.py

```diff
@@ -382,6 +382,10 @@ def get_service_schema():
             'replicas': {
                 'type': 'integer',
             },
+            'load_balancing_policy': {
+                'type': 'string',
+                'enum': ['round_robin']
```

Comment:
```suggestion
```
This would require 2 place to change when we add policy. Maybe we could just remove it and check it in the code?